doc/user/content/installation/install-on-aws/_index.md

```diff
@@ -294,9 +294,14 @@ components:
    https://github.com/MaterializeInc/terraform-aws-materialize?tab=readme-ov-file#input_materialize_instances)
    for the Materialize instance configuration options.
 
-   Starting in v0.4.0, a self-signed `ClusterIssuer` is deployed by default. The
-   `ClusterIssuer` is deployed on subsequent after the `cert-manager` is
-   running.
+   - **Starting in v0.4.0**, a self-signed `ClusterIssuer` is deployed by
```

Comment:
Depending on how prominent we want this config option, can actually remove.  If we envision users only setting this rarely, can remove to de-clutter the page since the same page has a [release section with the same info on it](https://preview.materialize.com/materialize/32544/self-managed/v25.1/installation/install-on-aws/#set-up-aws-kubernetes-environment-and-install-materialize).